backend/Wheelingful.API/Extensions/Endpoints/BookReaderExtension.cs

```diff
@@ -11,26 +11,18 @@ public static class BookReaderExtension
 {
     public static void MapBookReaderApi(this IEndpointRouteBuilder endpoints)
```

Comment:
as it is now books, probably merge 2 methods (MapBookReaderApi + MapBookAuthorApi)